doc/DeveloperGuide.md

```diff
@@ -104,3 +108,11 @@ Use case ends.
 ##### Private contact detail
 
 > A contact detail that is not meant to be shared with others
+
+##### Crashes
```

Comment:
is this necessary?

Reply:
Not necessary, better to put the explanation on the NFR itself.